docs/guide-ru/html-helper.md

```diff
@@ -0,0 +1,43 @@
+Html хелпер
```

Comment:
English docs should be updated. Are you OK doing it? If not, I can take this part.